[R2] Clarify Worker request body limits (#5014)
cloudflare/cloudflare-docs: content/r2/platform/limits.md

2. The max upload size is 5 MB less than 5 GB, so 4.995 GB.
3. Max upload size applies to uploading a file via one request, uploading a part of a multipart upload, or
copying into a part of a multipart upload. If you have a Worker, its inbound request size is
constrained by [Workers request limits](/workers/platform/limits#request-limits). The max upload size limit does not apply to subrequests.

> If you need to upload larger files, browse the [examples](/r2/examples/) on how to use the S3 API.

Reviewer: ```suggestion
Review the [Examples](/r2/examples/) on how to use SDKs with the S3 API to upload large files.
```
